packages/Maleo.js/src/build/index.ts

```diff
@@ -8,20 +8,20 @@ import { IBuildOptions } from '@interfaces/build/IBuildOptions';
 import { Context, CustomConfig, StaticPages } from '@interfaces/build/IWebpackInterfaces';
 import { buildStatic } from '@build/static/static';
 
-const cwd = process.cwd();
-
 export const getConfigs = (options: IBuildOptions): Configuration[] => {
-  const { env, buildType } = options;
+  const { env, buildType, minimalBuild, experimentalLazyBuild } = options;
 
   const context: Context = {
     env,
-    projectDir: cwd,
+    projectDir: process.cwd(),
```

Comment:
I think it is better to put the `process.cwd()` inside function, to make sure that everytime the function get called it always use the latest current working directory